src/UI/logmodel.cpp

```diff
@@ -2,12 +2,37 @@
 #include "logmodel.h"
 
 LogModel::LogModel(QObject *parent)
-    : QAbstractListModel(parent),
-      timer{new QTimer(this)}
+    : QAbstractTableModel(parent)
 {
-    timer->setInterval(100);
-    connect(timer, &QTimer::timeout , this, &LogModel::test);
-    timer->start();
+    LogInfo update{"C://Update.exe","Write",0,5,5,"None","handle","24:00:00","Angry User"};
+    logs.push_back(update);
+}
+
+QVariant LogModel::headerData(int section, Qt::Orientation orientation, int role) const
+{
+    if(role == Qt::DisplayRole && orientation == Qt::Horizontal){
+        switch(section){
+        case Columns::FilePath:
+            return "File Path";
+        case Columns::OperationType:
+            return "Operation Type";
+        case Columns::Pid:
+            return "PID";
+        case Columns::NumberOfBytes:
+            return "Number Of Bytes";
+        case Columns::Offset:
+            return "Offset";
+        case Columns::Preview:
+            return "Preview";
+        case Columns::FileHandle:
+            return "File Handle";
+        case Columns::OperationTime:
```

Comment:
I think this column should be at the beginning of the table.

Reply:
I moved FileHandle at the beginning.

Reply:
I meant the OperationTime.